cmd/vela-worker/operate.go

```diff
@@ -23,6 +26,35 @@ func (w *Worker) operate() error {
 	if err != nil {
 		return err
 	}
+	// Define the database representation of the worker
+	// and register itself in the database
+	registryWorker := new(library.Worker)
+	registryWorker.SetHostname(w.Config.Hostname)
+	registryWorker.SetAddress(w.Config.Server.Address)
+	registryWorker.SetRoutes(w.Config.Queue.Routes)
+	registryWorker.SetActive(true)
+	registryWorker.SetLastCheckedIn(time.Now().UTC().Unix())
+	w.register(registryWorker)
+	if err != nil {
+		logrus.Error("unable to register worker with the server")
+	}
```

Comment:
I think we should consider having the `register()` function return an `error` 👍 

I think we could then update this code to have `logrus` output that error.

That way, we can create a good error message in the `register()` function and have it output here 👍 

```suggestion
	w.register(registryWorker)
	if err != nil {
		logrus.Error(err)
	}
```